src/main/java/io/reactivex/plugins/RxJavaPlugins.java

```diff
@@ -446,6 +446,8 @@ public static Scheduler onNewThreadScheduler(@NonNull Scheduler defaultScheduler
      */
     @NonNull
     public static Runnable onSchedule(@NonNull Runnable run) {
+        ObjectHelper.requireNonNull(run, "runnable is null");
```

Comment:
Forgot to add explanation to PR why I've added check here and not in `Scheduler.schedule*()` methods:

Reason behind that is abstract `Scheduler` has 3 overloads, one of them reuses another one + any concrete scheduler can override any of those methods and call other overloads, so as a result I had checks duplicated in some cases.

This `RxJavaPlugins` call is done by all schedulers before actual schedule/run of the original `Runnable` so it seems to be a good common point that only executed once and guarantees now that `Runnable` is not null for both user defined `RxJavaPlugins.onSchedule()` function and all RxJava schedulers.